bin/build-plugin-zip.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+TMP_BUILD_DIR=/tmp/gutenberg
+
+# Build the assets
+npm install
+npm run build
+
+# Prepare the plugin directory
+rm -rf $TMP_BUILD_DIR
+mkdir -p $TMP_BUILD_DIR/i18n/build
```

Comment:
Minor quality of life tip: You can create multiple directories in one go like so:

```bash
mkdir -p $TMP_BUILD_DIR/{i18n,blocks,editor,element}/build
```